src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -935,7 +936,7 @@ internal class ClusterCoreDaemon : UntypedActor, IRequiresMessageQueue<IUnbounde
         /// TBD
         /// </summary>
         protected readonly UniqueAddress SelfUniqueAddress;
-        private const int NumberOfGossipsBeforeShutdownWhenLeaderExits = 3;
+        private const int NumberOfGossipsBeforeShutdownWhenLeaderExits = 5;
```

Comment:
What was the purpose of increasing this value? Not questioning the decision to do it, but just want to understand the reason for the change.